clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -8651,6 +8651,11 @@ class DefaultedComparisonAnalyzer
         assert(Best->BuiltinParamTypes[2].isNull() &&
                "invalid builtin comparison");
 
+        // FIXME: If the type we deduced is a vector type, we mark the
+        // comparison as deleted because we don't yet support this.
+        if (isa<VectorType>(T))
+          return Result::deleted();
```

Comment:
The other versions of 'deleted' all seem to diagnose first, is there a reason to not do so here?

Reply:
Oh, I missed the "explain deleted" code paths entirely. I'll address that.